### PR TITLE
Changing a string for specific translation for air spawn

### PR DESCRIPTION
**Pull request by rlnt**
Changed STR_DN_AIR to STR_ESS_AIR because of new stringtable and specific translations for ESS.

### CODE REVIEW

**Comment by ebayShopper** [comment]
Both the German and English are the same as the A2OA translation:
```
<Key
      ID="str_dn_air">
      <Original>Air</Original>
      <French>Aérien</French>
      <Spanish>Aire</Spanish>
      <Italian>D’aria</Italian>
      <Hungarian>Légi</Hungarian>
      <Polish>Lotnictwo </Polish>
      <Russian>Авиация</Russian>
      <German>Luft</German>
      <Czech>Letectvo</Czech>
      <English>Air</English>
      <Japanese>空気</Japanese>
    </Key>
```
Arma 2 Operation Arrowhead\Expansion\dta\languagecore\stringtable.xml

**Comment by rlnt** [comment]
Normally I would prefer something like Halo instead of air, that's why in my opinion a specific own string would be better, but that's your choice and I agree with you.

**Comment by ebayShopper** [comment]
Go ahead and change it to "HALO" if you want. That's fine with me. I don't see any stock A2 translation for that.

File: stringtable.xml [hunk 1]
```diff
@@ -49,5 +49,9 @@
 			<English>Ground</English>
 			<German>Boden</German>
 		</Key>
+		<Key ID="STR_ESS_AIR">
+			<English>Halo</English>
```
Comment (ebayShopper) [inline stringtable.xml]: English should be all caps HALO since it is an abbreviation.

Comment (DAmNRelentless) [inline stringtable.xml]: agree

**Pull request merged**